### ISSUE TITLE
linux kernel support

### ISSUE THREAD

**Issue by nickdesaulniers** [M680x0/M680x0-mono-repo#18]
Hello, at [clangbuiltlinux.github.io](https://clangbuiltlinux.github.io/), we're working on supporting the Linux kernel with LLVM. m68k would be a huge deal for us so it's really cool to see this work you all have been doing.  If there's any interest in trying to build a working Linux kernel image, please let us know; we'd like to help.

**Comment by glaubitz** [comment]
Of course we are interested. Do you maybe want to give a talk on that topic? If yes, you can join our next m68k online meeting: http://m68k.info.

**Comment by nickdesaulniers** [comment]
Definitely!  Can you please ping this thread when there's a date scheduled for the next meeting?

**Comment by glaubitz** [comment]
I will :).

**Comment by nickdesaulniers** [comment]
adding a link to issues tagged with m68k: https://github.com/ClangBuiltLinux/linux/labels/%5BARCH%5D%20m68k cc @mshockwave

**Comment by glaubitz** [comment]
@nickdesaulniers The AsmParser was merged in LLVM today, so I guess that's a big step forward towards building the kernel with clang on m68k.

**Comment by nickdesaulniers** [comment]
Nice; we're going to do a bug scrub tomorrow from 9am-5pm pacific time: https://meet.google.com/yvj-tmcy-jup. Hopefully we'll have more bugs filed or closed out.

**Comment by nickdesaulniers** [comment]
Consider stopping by!

**Comment by glaubitz** [comment]
Sounds good! I'll be there.

**Comment by nickdesaulniers** [comment]
Thanks for stopping by @glaubitz and @mshockwave !
